nirum_wsgi.py

```diff
@@ -298,6 +367,29 @@ def _raw_response(self, status_code, response_json, **kwargs):
         return Response(content, status_code, headers, **kwargs)
 
 
+def compile_uri_template(template):
+    if isinstance(template, bytes):
+        template = template.decode()
```

Comment:
If https://github.com/spoqa/nirum/pull/197 is merged this condition can be removed.